src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -394,9 +394,13 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
     async function watchFeedback() {
       const answer = await getAnswerMetadata(interactiveId);
       if (answer?.id) {
-        unsubscribe = watchQuestionLevelFeedback(answer.id, (fb) => {
-          setFeedback(fb);
-        });
+        unsubscribe = watchQuestionLevelFeedback((fbs) => {
```

Comment:
It looks like you forgot to return a function that calls `unsubscribe` here.

Reply:
Never mind - my bad, I didn't expand the diff context enough to see where you returned it. :(